ingest/Snakefile

```diff
@@ -0,0 +1,87 @@
+from snakemake.utils import min_version
+
+min_version(
```

Comment:
Minimum might actually be higher now that we use a default profile - something like 7.30

Below one might possibly need to pass in extra cli flags